pallets/parachain-staking/src/inflation.rs

```diff
@@ -34,15 +33,15 @@ pub struct RewardRate {
 }
 
 /// Convert annual reward rate to per_block.
-fn annual_to_per_block(rate: Perquintill) -> Perquintill {
-	rate / YEARS
+fn annual_to_per_block<T: Config>(rate: Perquintill) -> Perquintill {
```

Comment:
I guess we don't need the generic `T` anymore in this function.

Reply:
fixed in b03f3c6